src/NuGetPackageVerifier/build/NuGetPackageVerifier.targets

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Target Name="VerifyPackages"
+          AfterTargets="Pack"
```

Comment:
> The goal I'm working towards an MSBuild version of KoreBuild that has many of the same features of KoreBuild

Maybe we should run this by @anurse. @anurse started work on an alternate build system https://github.com/aspnet/KoreBuild/tree/dev/src/Microsoft.AspNetCore.Build / https://github.com/aspnet/BuildTrain/tree/train/1.0.0 a while ago but we never got around to switching to it. I'd rather we not invest in any work until we have some consensus from the engineering team on what we want here.